pkg/network/node/iptables.go

```diff
@@ -273,6 +273,28 @@ func (n *NodeIPTables) getNodeIPTablesChains() []Chain {
 		},
 	)
 
+	// Don't track vxlan traffic with conntrack, as it is not needed and decreases performance
+	chainArray = append(chainArray,
+		Chain{
+			table:    "raw",
+			name:     "OPENSHIFT-VXLAN-NOTRACK",
```

Comment:
assuming we do this, I'd make the table be `OPENSHIFT-NOTRACK` or even `OPENSHIFT-RAW-PROCESSING` or something like that (I don't really like "processing"). Something slightly more generic, in case we want to use it for more things later.